tests/zeek/udp.zeek

```diff
@@ -17,7 +17,8 @@ public type Message = unit {
 # @TEST-START-FILE udp-test.evt
 protocol analyzer spicy::UDP_TEST over UDP:
     parse with UDPTest::Message,
-    port 31337/udp;
+    port 11337/udp-11340/udp,
+    ports {31337/udp-31340/udp};
```

Comment:
I'm confused about syntax now: are these two lines doing the same?

Reply:
Yes, they are both supported (but they construct different ranges here).

On the implementation side this is possible by using identical code to extract both a single and multiple ports (here: `extract_ports` which always returns a vector of `Port`).

Reply:
Ok, the syntax remains a bit non-intuitive to me, but with the docs it's fine.